### DISCUSSION TITLE
Translate Question

### DISCUSSION THREAD

**Discussion by davi19** [starship/starship#2759] [Q&A] [answered]
Hi, i¨m working in translate to Brasilian Portuguese and my question is when the links with the translate go to production ? i´m have make a PR or not ?

**Comment by matchai** [comment, marked as answer]
Crowdin automatically creates PRs containing translations.
You can see the latest one here: https://github.com/starship/starship/pull/2733

**Reply by davi19** [reply]
Ok thanks.
